### DISCUSSION TITLE
Not able to import SCSS module in project

### DISCUSSION THREAD

**Discussion by NaveenReddy1** [vitejs/vite-plugin-vue#449] [Q&A]
### Related plugins

- [X] [plugin-vue](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue)

- [ ] [plugin-vue-jsx](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue-jsx)

### Describe the bug

Hi i am trying to import the scss files in my project but scss classes are not applying.

Please correct/suggest me on this.

### Reproduction

https://stackblitz.com/edit/vitejs-vite-rwnnns?file=package.json

### System Info

```shell
Vue: 3.4
vite: 5.2
sass: 1.75.0
```

### Used Package Manager

pnpm

**Comment by edison1105** [comment]
in App.vue
```html
<style scoped lang="scss">
```
should be
```html
<style lang="scss">
```
